aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,536 @@
+
+NuGet Packages
+--------------
+
+Getting the core components added to the project via nuget can be done using either the PowerShell command, modifying the package.json file directly, adding the package to the dependencies node, or by right clicking the references folder and selecting "Manage NuGet Packages" and searching for "Swashbuckle". Be sure to select the "Include Pre-Release" to get the latest version while in beta.
+
+*Install using NuGet*
+
+.. code-block:: bash
+
+    Install=Package Swashbuckle -Pre
+    
+*Install by modifying project.json*
+
+.. code-block:: javascript
+   :emphasize-lines: 17
+
+    "dependencies": {
+      "Microsoft.NETCore.App": {
+        "version": "1.0.0",
+        "type": "platform"
+      },
+      "Microsoft.ApplicationInsights.AspNetCore": "1.0.0",
+      "Microsoft.AspNetCore.Mvc": "1.0.0",
+      "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0",
+      "Microsoft.AspNetCore.Server.Kestrel": "1.0.0",
+      "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0",
+      "Microsoft.Extensions.Configuration.FileExtensions": "1.0.0",
+      "Microsoft.Extensions.Configuration.Json": "1.0.0",
+      "Microsoft.Extensions.Logging": "1.0.0",
+      "Microsoft.Extensions.Logging.Console": "1.0.0",
+      "Microsoft.Extensions.Logging.Debug": "1.0.0",
+      "Microsoft.Extensions.Options.ConfigurationExtensions": "1.0.0",
+      "Swashbuckle": "6.0.0-beta901"
+
+
+*Install via Manage NuGet Packages*
+
+.. image:: web-api-help-pages-using-swagger/_static/manage-nuget-packages.png
+
+
+*Manage NuGet Packages Tab*
+
+.. image:: web-api-help-pages-using-swagger/_static/add-swagger.png
+
+
+Adding Middleware
+-----------------
+
+After adding the NuGet package to the project, the Startup.cs class needs to modified for the Swagger components to available for use.
+
+First, add SwaggerGen to the services collection in the Configure method.
+
+.. code-block:: c#
+   :emphasize-lines: 12
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        // Add framework services.
+        services.AddMvc();
+
+        services.AddLogging();
+
+        // Add our repository type
+        services.AddSingleton<ITodoRepository, TodoRepository>();
+
+        // Inject an implementation of ISwaggerProvider with defaulted settings applied
+        services.AddSwaggerGen();
+    }
+
+Next, in the Configure method, enable the middleware for serving generated JSON document and the SwaggerUI. 
+
+.. code-block:: c#
+   :emphasize-lines: 6,9
+
+    public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
+    {
+        app.UseMvcWithDefaultRoute();
+
+        // Enable middleware to serve generated Swagger as a JSON endpoint
```

Comment:
Code comments are OK but they need to use correct punctuation(end in period here) and most importantly you need to add them to the article. Once this is translated, the code comments will ***NOT*** be translated.